README.md

```diff
@@ -91,9 +89,7 @@ from the model(s), it uses a view to *render* an HTML page.
 
 6. The rendered view is then sent back to the client as a response.
 
```

Comment:
you do - check out the tunr solution - what's the model? what's the controller? whtat's the view?